pytorch_lightning/lite/lite.py

```diff
@@ -468,6 +505,19 @@ def _supported_device_types() -> Sequence[_AcceleratorType]:
             _AcceleratorType.TPU,
         )
 
+    @staticmethod
+    def _supported_strategies() -> Tuple[Type[Strategy], ...]:
+
+        return (
+            SingleDeviceStrategy,
```

Comment:
Bagua is not supported?

Reply:
I am not sure. It probably hasn't been tested yet. I kept these as matches from the _StrategyTypes @awaelchli put there initially.

Reply:
cc: @awaelchli

Reply:
Yes, in theory Lite would support it since it is a modification of the DDPStrategy. However, we should sanity check this first before we add it to the types here. I created an issue for this: #12500